Content.Client/Overlays/ImageOverlay.cs

```diff
@@ -11,13 +11,11 @@ namespace Content.Client.Overlays;
 /// </summary>
 public sealed partial class ImageOverlay : Overlay
 {
-    [Dependency] private readonly IPlayerManager _playerManager = default!;
-    [Dependency] private readonly IEntityManager _entityManager = default!;
     [Dependency] private readonly IResourceCache _resourceCache = default!;
 
     public override OverlaySpace Space => OverlaySpace.WorldSpace;
 
-    public readonly List<(ShaderInstance, ImageShaderValues)> ImageShaders = new();
+    public readonly List<(ShaderInstance, ImageShaderValues)> TupleOfImageShaders = new();
```

Comment:
What I meant specifically about the tuple was declaring it like `public readonly List<(ShaderInstance Instance, ImageShaderValues ShaderValues)> TupleOfImageShaders = new();` While it doesn't actually come up in your code, this makes it so you can just do `ImagesShaders[0].Instance` instead of `ImageShaders[0].Item1`.